test/component/summary/testWillSection.js

```diff
@@ -59,12 +57,10 @@ describe('summary-will-section', () => {
                     const playbackData = {};
                     playbackData.willLeft = willContent.left.question;
                     playbackData.willOriginal = willContent.original.question;
-                    playbackData.date = willContent.date.question;
-                    playbackData.dateQuestion = willContent.date.willDateQuestion;
                     playbackData.willCodicils = willContent.codicils.question;
                     playbackData.applicantExecutor = applicantContent.executor.question;
 
-                    Object.assign(playbackData, willData.will, willData.applicant);
+                    Object.assign(playbackData, willData.applicant);
```

Comment:
I don't think we want to remove `willData.will`, rather we need to remove the now unused test data from `test/data/will.json`.